src/utils/fileUtils.ts

```
@@ -0,0 +1,41 @@
+    try {
+      await fs.writeFile('LICENSE', content);
+      return true;
+    } catch (error) {
+      throw new Error(
+        `Failed to write LICENSE file: ${
+          error instanceof Error ? error.message : String(error)
+        }`,
+      );
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)